VstsTasks/AzureDtlCreateVM/task-funcs.ps1

```diff
@@ -0,0 +1,180 @@
+function Handle-LastError

```

Comment:
You have duplication of some methods, anyway to have a shared func file between the folders?